src/main/java/com/revature/lostchapterbackend/controller/BookController.java

```diff
@@ -0,0 +1,48 @@
+package com.revature.lostchapterbackend.controller;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.revature.lostchapterbackend.model.Book;
+import com.revature.lostchapterbackend.service.BookService;
+
+@RestController
+public class BookController {
```

Comment:
Need @CrossOrigin(originPatterns = "*", allowCredentials = "true")

Reply:
I will tell the display/search product feature team to add this for future merge, would that be fine?

Reply:
Sounds good to me.